features/css-logical-properties.md

```diff
@@ -0,0 +1,12 @@
+---
+title: CSS Logical Properties and Values
+category: css, layout
+bugzilla: 1323940
+firefox_status: in-development
+spec_url: https://drafts.csswg.org/css-logical/
+caniuse_ref: css-logical-props
+chrome_ref: 6237096230518784
+ie_ref: csslogicalpropertieslevel1
```

Comment:
broken IE ref. That is the slug, not the name...

Reply:
I think I've fixed this now...